tgstation.dme

```diff
@@ -23,6 +23,7 @@
 #include "code\__DEFINES\_protect.dm"
 #include "code\__DEFINES\_tick.dm"
 #include "code\__DEFINES\access.dm"
+#include "code\__DEFINES\nitrile.dm"
```

Comment:
Don't edit this file manually, otherwise every PR from this point forward is going to have this auto-fixed.

Reply:
you can edit it manually you just need to sort your includes